flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adapter/DefaultExecutionTopology.java

```diff
@@ -130,7 +131,8 @@ public EdgeManager getEdgeManager() {
         return edgeManager;
     }
 
-    public static DefaultExecutionTopology fromExecutionGraph(ExecutionGraph executionGraph) {
+    public static DefaultExecutionTopology fromExecutionGraph(
+            DefaultExecutionGraph executionGraph) {
```

Comment:
Why do we need this change? Or should this commit be ahead of the commit to "Optimize the initialization of DefaultExecutionTopology"?

Reply:
> Why do we need this change? Or should this commit be ahead of the commit to "Optimize the initialization of DefaultExecutionTopology"?

Because `EdgeManager` is located at `InternalExecutionGraphAccessor` interface rather than `ExecutionGraph` interface. To get `EdgeManager`, I'd prefer to pass the `DefaultExecutionGraph` as the parameter here.

Reply:
I see. So this is a fix commit. It's better to clearly mark it out for future merging.